app/views/pages/bt_wifi_privacy_notice.html.erb

```diff
@@ -0,0 +1,81 @@
+    </p>
+    <p class="govuk-body">
+      As part of the scheme, DfE, as a data controller, provides your school or local authority the following information:
+      <ul class="govuk-list govuk-list--bullet">
+        <li>a user login name and password</li>
+      </ul>
+    </p>
+    <p class="govuk-body">
+      This information is then provided to you to enable BT WiFi access to the internet. The access is restricted in order to keep children safe when online.
+    </p>
+    <p class="govuk-body">
+      When your school or local authority provides you with the user login name and password, it does not share your name or any other personal data about you with the DfE.
+    </p>
+    <h2 class="govuk-heading-l">
+      Why we can use personal information
+    </h2>
+    <p class="govuk-body">
+      To use the personal information above, we need to meet one (or more) conditions in the data protection legislation. For providing this service we use personal information as part of our legitimate interest.
+    </p>
+    <h2 class="govuk-heading-l">
+      How long we hold personal information
+    </h2>
+    <p class="govuk-body">
+      Any personal information will only be held as long as necessary and no later than 31 August 2021.
+    </p>
+    <h2 class="govuk-heading-l">
+      Who we share your personal information with
+    </h2>
+    <p class="govuk-body">
+      The user login name and password is supplied to the school or local authority but they do not share your name with us, the DfE.
+    </p>
+    <h2 class="govuk-heading-l">
+      Your individual rights
+    </h2>
+    <p class="govuk-body">
+      Under the Data Protection Act 2018, you are entitled to ask if we hold information relating to you and ask for a copy, by making a ‘subject access request’.
+    </p>
+    <p class="govuk-body">
+      For further information and how to request your data, please use the <a class="govuk-link" href="https://www.education.gov.uk/help/contactus">contact form</a> in the Personal Information Charter at
+    </p>
+    <p class="govuk-body">
+      <a class="govuk-link" href="https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter ">https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter</a>
+    </p>
+    <p class="govuk-body">
+      under the ‘How to find out what personal information we hold about you’ section.
+    </p>
+    <p class="govuk-body">
+      You can read further information about your data protection rights at the <a href="https://ico.org.uk/" class="govuk-link">Information Commissioner’s Office</a>.
+    </p>
+  </div>
+</div>
+
+<!--
+
+
+#
+
+
+For further information and how to request your data, please use the ‘contact form’ in the Personal Information Charter at
+
+https://www.gov.uk/government/organisations/department-for-education/about/personal-informa1tion-charter
```

Comment:
is there a typo in `personal-informa1tion-charter` ?

Reply:
I guess it's a comment, so not a blocker

Reply:
oops 😬  – that whole thing needs to go. fixed and force pushed